GCRealTimeMon.sln

```diff
@@ -31,6 +33,14 @@ Global
 		{A5A83224-3B9E-4E6C-9E1C-7C286170157E}.Release|Any CPU.Build.0 = Release|Any CPU
 		{A5A83224-3B9E-4E6C-9E1C-7C286170157E}.Release|x64.ActiveCfg = Release|x64
 		{A5A83224-3B9E-4E6C-9E1C-7C286170157E}.Release|x64.Build.0 = Release|x64
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Debug|x64.ActiveCfg = Debug|x64
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Debug|x64.Build.0 = Debug|x64
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Release|Any CPU.Build.0 = Release|Any CPU
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Release|x64.ActiveCfg = Release|x64
+		{FB7F07EE-1473-4E5D-9996-B428188E4268}.Release|x64.Build.0 = Release|x64
```

Comment:
the active dotnet-gcmon proj should be release, I think in VS terms this means you only have the lines that say release? right now if I open this sln in VS this is built as debug.

Reply:
Baed on you choice in VS combobox, the default configuration is kept in the .suo file that should not be in source control.
I think it is good to keep the Debug config in case you want to debug it but no problem to remove it.
Also it will be unexpected to have release in the Configuration Manager UI even if debug is selected as the current configuration
![image](https://user-images.githubusercontent.com/10163781/142363025-ded8662f-e8a2-487b-87c9-aff9167bb444.png)
Let me know what is your call